scrapy/downloadermiddlewares/retry.py

```diff
@@ -33,12 +33,12 @@
 
 class RetryMiddleware:
 
-    # IOError is raised by the HttpCompression middleware when trying to
+    # OSError is raised by the HttpCompression middleware when trying to
```

Comment:
Was this fixed? I'm not sure if the test changed here: https://github.com/scrapy/scrapy/pull/4716/files#r466436261 fixed this. 

Even if it's not fixed, do you think we should remove this comment or keep it?

Reply:
I think leaving the comment makes sense. I think it’s the only exception with a comment because it is the only exception for which it is not obvious why it belongs in `EXCEPTIONS_TO_RETRY`.

I do wonder if there’s any potential for backward incompatibility here, and whether we can address it or we can only document it. The best I can think of is a subclass which rewrites the tuple to remove `IOError` and leave all other exceptions, which could be implemented in a way that would fail to remove `OSError`, but I do not know how we could address that.